packages/tasks/src/library-ui-elements.ts

```diff
@@ -767,8 +767,8 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: Partial<Record<ModelLibraryKey, Librar
 		docsUrl: "https://huggingface.co/docs/hub/stable-baselines3",
 		snippets: stableBaselines3,
 	},
-	"ml-agents": {
-		btnLabel: "ml-agents",
+	"unity-ml-agents": {
+		btnLabel: "unity-ml-agents",
```

Comment:
Can we change the link below (`repoUrl`) to the actual Unity repo rather than the fork? I think they merged the changes

Reply:
✅